lib/package.js

```diff
@@ -180,13 +130,13 @@ module.exports = function pkg(datastar, models) {
     }, {});
 
     return replica;
-  };
+  }
+
 
-  //
   // Deserialize some of our custom data structures stored as text until we know
   // the schema will enforce these for us nicely
-  //
-  Pkg.deserialize = function deserialize(results) {
+
+  function deserialize(results) {
```

Comment:
add this as an extra function passed to `Dynastar` but modify it to only support accepting a single object. we will need to integrate this in places where we expected the deserializing to happen previously in the application